src/NuGet.Core/NuGet.Build.Tasks.Pack/PackTaskLogic.cs

```diff
@@ -346,6 +347,27 @@ private void PopulateFrameworkAssemblyReferences(PackageBuilder builder, IPackTa
                             )));
         }
 
+        private void PopulateFrameworkReferences(PackageBuilder builder, IPackTaskRequest<IMSBuildItem> request)
+        {
+            var tfmSpecificRefs = new Dictionary<string, ISet<string>>(StringComparer.OrdinalIgnoreCase);
+
+            foreach (var tfmRef in request.FrameworkReferences)
+            {
+                var targetFramework = tfmRef.GetProperty("TargetFramework");
```

Comment:
Is there any scenario where this property might not exist?  What is the behavior of `GetProperty(…)` when the property does not exist?  Will `GetProperty(…)` throw?  If it returns `null`, are the subsequent statements below resilient to `null`?

Reply:
That should not happen as we control the IMSBuildItem generation.